src/dotnet/APIView/APIViewWeb/web.config

```diff
@@ -1,15 +1,33 @@
 <configuration>
-  <system.webServer>  
-    <rewrite>  
-        <rules>  
-          <rule name="Redirect requests to default azure websites domain" stopProcessing="true">
-            <match url="(.*)" />  
-            <conditions logicalGrouping="MatchAny">
-              <add input="{HTTP_HOST}" pattern="^apiview\.azurewebsites\.net$" />
-            </conditions>
-            <action type="Redirect" url="http://apiview.dev/{R:0}" />  
-          </rule>  
-        </rules>  
-    </rewrite>  
-  </system.webServer>  
-</configuration>  
\ No newline at end of file
+  <system.webServer>
+    <httpProtocol>
+      <customHeaders>
+        <add name="Access-Control-Allow-Origin" value="https://spa.apiviewuxtest.com" />
+        <add name="Access-Control-Allow-Methods" value="*" />
+        <add name="Access-Control-Allow-Headers" value="Content-Type" />
+        <add name="Access-Control-Request-Headers" value="*" />
+        <add name="Access-Control-Allow-Credentials" value="true" />
+      </customHeaders>
+    </httpProtocol>
+    <rewrite>
+      <rules>
+        <rule name="Redirect requests to default UI instance domain" stopProcessing="true">
+          <match url="(.*)" />
+          <conditions logicalGrouping="MatchAny">
+            <add input="{HTTP_HOST}" pattern="^apiviewstaging\.azurewebsites\.net$" />
+          </conditions>
+          <action type="Redirect" url="https://apiviewuxtest.com/{R:0}" />
+        </rule>
+        <rule name="Rewrite root SPA requests">
+          <match url="^((?!spa/).*)$" />
+          <conditions logicalGrouping="MatchAll">
+            <add input="{HTTP_HOST}" pattern="^spa\.apiviewuxtest\.com$" />
```

Comment:
Don't we need to remove this condition to get this working for prod and production staging slot?

Reply:
Just need to add another rule to handle the production instance.